tests/scripts/ci.sh

```diff
@@ -4,5 +4,5 @@ set -e
 set -u
```

Comment:
at some point we should move this script into a circle command, it just proxy calls two scripts, and just adds one more un-needed level.